homeassistant/components/plugwise/__init__.py

```diff
@@ -148,6 +161,38 @@ def available(self):
         """Return True if entity is available."""
         return self._coordinator.last_update_success
 
+    @property
+    def name(self):
+        """Return the name of the entity, if any."""
+        if not self._name:
+            pass
```

Comment:
```suggestion
            return None
```

It might make more sense to return a default name since we know its a "Smile Gateway"